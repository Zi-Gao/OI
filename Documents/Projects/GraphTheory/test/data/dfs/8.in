656 821
153 436 6882
158 535 4527
492 655 349
255 241 23014
5 219 18624
565 74 6407
158 84 14739
597 293 24712
23 253 21053
445 597 27098
146 158 31649
128 445 22571
220 106 21132
102 571 1154
179 113 1251
422 535 4123
516 276 4822
384 620 26383
532 419 9086
412 28 20899
515 221 22957
508 211 15586
531 105 231
519 422 4931
539 177 21424
69 510 25740
522 642 21430
326 414 18482
102 391 383
315 383 26599
566 6 1195
101 236 2070
647 640 22242
383 476 17013
617 422 4865
226 446 2818
587 471 30516
507 470 4601
340 15 17393
44 468 13744
208 204 15124
133 47 3412
145 386 27017
163 27 12799
312 607 27470
341 16 23969
45 496 19631
297 114 20398
605 47 23815
406 596 13715
570 336 1648
558 627 10586
553 67 23335
524 277 17165
494 265 21645
625 157 18869
125 374 2315
611 637 22274
279 69 13639
497 153 19720
153 369 10859
641 22 24463
342 136 22122
234 168 15083
75 350 31880
55 92 4182
222 449 9111
11 376 5186
109 9 3226
147 117 21789
69 152 32726
196 155 20858
591 7 14201
108 529 27302
427 129 30299
594 48 19716
500 443 16009
589 289 14800
277 221 28310
115 123 22034
25 564 7658
160 465 20718
615 83 12161
64 86 26513
127 492 4064
436 56 5991
71 309 15909
189 413 16230
197 111 27007
264 642 6030
315 50 25454
546 371 20972
346 25 15515
286 485 31427
260 52 25904
213 65 21932
95 68 27375
552 230 5065
504 116 15993
353 578 25438
491 625 5771
504 195 4970
68 594 25368
42 180 26144
316 642 7042
351 472 17225
178 95 23149
429 551 22516
615 444 32468
358 631 25350
82 643 15730
161 92 1663
260 368 32409
577 578 920
181 183 25609
417 255 23552
598 342 12143
518 522 31973
570 525 19847
467 315 26766
300 381 21285
448 432 22950
107 550 4225
176 428 10133
636 490 21671
538 178 21881
139 631 24022
655 349 15763
19 361 5420
301 19 18759
570 450 25341
168 220 10766
550 199 30366
364 107 8664
301 494 6032
538 621 10712
641 586 20297
551 514 15540
421 144 16919
12 189 17798
45 132 12999
112 524 18236
531 145 16409
90 43 18477
307 85 15403
253 116 31752
583 217 23904
474 190 6751
263 518 17772
31 212 23170
578 255 18059
184 220 15717
166 332 14355
410 53 13506
293 593 25306
393 159 25277
499 366 25384
452 276 12204
337 511 8500
248 21 23510
599 531 601
72 74 4424
68 260 1148
313 283 24074
9 634 25195
171 146 31073
598 70 31509
37 118 22236
443 239 20198
84 180 24672
137 450 19891
408 58 19570
337 429 13563
116 289 30825
610 199 12644
262 558 1620
111 277 10168
166 473 19638
441 135 18653
504 174 31334
233 611 12655
588 471 15156
510 526 8457
652 343 14883
420 395 7779
362 436 9307
127 230 29902
301 648 23579
278 255 11444
107 245 27986
416 616 17442
354 574 7599
209 526 19853
377 530 12355
484 636 2638
376 316 14602
93 539 25483
105 120 14074
649 487 8065
261 474 11106
111 117 10863
30 515 7108
108 265 4503
34 499 26480
629 292 13319
281 432 11398
496 618 42
403 108 12333
417 334 9817
344 562 27676
176 275 18394
112 368 6094
521 250 10001
277 241 19955
22 105 2264
652 109 24320
562 445 8237
253 103 2025
645 533 21688
377 218 12391
527 110 18918
588 357 5732
511 302 3960
409 96 29781
60 396 9696
84 481 20144
535 471 20427
15 402 14535
241 563 22421
56 57 7070
58 64 19114
326 50 27044
239 75 1534
519 105 16473
351 499 5660
83 421 21816
291 63 12931
246 406 15754
208 265 20293
313 328 14574
214 273 12581
499 255 13487
114 13 28589
276 105 21069
41 430 11753
139 653 26040
580 87 24017
296 129 12341
295 368 27527
414 411 8931
189 306 1263
418 295 16474
49 607 10067
128 503 22233
581 631 13
521 510 21759
223 181 20075
461 467 30984
156 301 9290
202 505 7927
118 80 28705
251 187 3138
242 101 30313
9 411 22489
313 13 6570
115 418 919
115 506 22427
356 597 6274
160 101 1800
41 486 10606
356 382 30091
447 123 6096
54 178 21812
69 182 25834
453 408 18863
38 618 19868
505 199 8592
274 366 31099
291 650 13260
422 281 13816
197 80 17035
223 251 7362
191 50 31690
341 289 12458
460 514 1429
514 50 32426
484 351 18106
254 285 14567
334 5 7749
444 642 8131
481 376 19237
50 487 29012
260 468 13000
27 290 4577
530 257 20630
522 352 16441
4 567 12684
305 481 32501
468 624 18025
137 316 30727
11 149 23597
44 494 4650
46 570 32213
58 232 18444
1 192 13215
216 584 8003
608 264 21303
470 380 29044
598 5 12700
496 279 9267
400 368 21153
561 261 31875
396 211 31568
56 93 253
81 635 9123
32 114 27152
313 585 31043
62 605 462
631 607 18519
428 450 7492
232 18 134
36 6 5976
515 41 31666
606 37 31124
83 521 31009
346 367 22897
151 640 15113
421 506 28924
532 383 11707
389 187 12267
439 254 18145
11 112 32381
450 3 12426
348 475 15797
284 171 9979
313 593 18648
288 473 381
441 604 27186
500 68 32159
33 347 27019
469 281 24999
352 147 9433
47 122 13864
298 328 5132
531 133 32751
47 582 16141
414 7 23248
138 213 6468
301 278 27381
24 649 30782
623 231 7612
24 561 846
631 382 29767
291 412 6208
478 266 24906
381 50 17777
527 415 28868
595 583 13290
159 339 19353
320 101 15450
26 173 591
645 278 30706
164 526 12633
590 325 31389
329 153 22200
104 3 5640
628 215 25557
521 397 7534
130 586 2022
4 179 12336
631 655 11837
544 64 9627
521 221 2377
374 586 24164
255 534 3640
564 610 4301
516 551 20095
2 592 10858
307 448 29609
1 223 12031
58 440 5956
53 193 7666
594 449 5057
609 79 28733
468 30 3231
607 610 16114
463 479 6981
517 46 9889
72 311 6985
563 34 5702
197 147 19488
160 202 30859
78 3 14177
440 380 7298
396 233 21035
623 611 32544
7 488 30029
163 183 10522
517 92 23354
30 14 19716
165 87 9203
457 188 16309
47 8 15979
539 228 22875
1 386 21529
229 446 22477
543 621 16042
400 488 2812
396 235 31000
120 518 31371
130 622 13448
303 385 24567
63 2 3783
192 126 21217
595 194 11069
585 266 27977
207 542 28838
138 275 3047
449 423 15610
272 628 13022
98 459 20041
355 641 6522
389 596 14964
645 381 32711
586 639 26887
135 366 18195
414 553 27053
159 399 25276
87 242 1764
396 375 10468
542 258 26628
152 341 25300
650 617 18107
488 354 19256
300 358 31615
582 143 31874
369 9 16815
514 93 15372
401 415 24407
156 471 7891
454 640 9682
343 333 19176
35 274 22468
499 76 9446
319 522 13846
180 193 6915
522 87 6409
301 344 1963
500 252 23412
363 528 25495
237 265 19907
205 401 17375
182 475 13147
481 576 31360
121 580 18542
516 172 12358
254 275 18268
507 287 1307
3 432 11320
623 396 16427
5 33 23058
443 109 27485
538 222 6409
309 182 28181
64 515 30038
456 615 26930
162 372 9938
645 286 19146
6 396 9635
270 149 23064
419 344 15380
239 318 27785
542 628 3782
243 164 6907
133 119 2996
35 591 14978
403 333 17866
177 414 7761
149 469 17918
196 44 20633
253 161 7202
486 84 3705
256 484 18678
304 421 30555
91 10 18456
492 201 6736
640 61 23526
518 270 27713
24 424 5749
34 387 18453
202 459 16931
173 499 30243
117 518 4465
242 442 27262
257 472 12578
632 211 510
346 148 18652
389 446 5146
307 451 30139
56 357 17386
106 235 10126
249 57 9449
467 505 21692
455 369 6229
20 434 523
482 389 20025
389 637 29361
385 447 8581
86 445 19066
604 284 10435
423 80 20209
57 276 23884
525 41 5575
104 528 14597
84 3 7660
583 326 12044
480 82 24087
290 59 32226
401 653 3763
136 534 6638
353 9 11884
195 467 25975
539 282 18376
326 432 10078
281 439 9112
151 17 17471
585 615 10023
579 283 13769
360 603 28262
92 228 10857
576 638 4500
155 556 11162
110 236 17158
212 135 13176
257 118 23846
610 90 19057
473 286 25018
116 268 11293
622 7 8979
451 626 7845
23 578 9406
415 195 10832
243 654 1213
397 530 18173
627 293 16252
485 342 29621
495 75 12285
431 294 4265
231 416 6896
201 280 20013
374 383 11951
225 126 9094
605 618 31576
226 558 5817
504 573 19837
625 524 14994
588 510 23327
227 317 14999
324 586 9711
373 142 9389
521 340 9995
331 496 28311
77 71 6067
362 362 2450
289 468 18342
304 305 16573
73 653 15657
584 201 12052
340 496 9193
526 550 24604
538 102 10034
507 115 15004
374 524 2351
132 65 14065
333 359 27705
440 168 10978
95 70 9304
655 171 13054
206 325 22145
610 344 1514
406 175 30779
190 504 22569
121 327 15792
347 30 30530
350 609 32081
379 211 31061
39 444 22427
75 312 23161
9 203 22175
222 389 13182
158 58 3156
222 95 28130
121 556 18404
339 35 1819
375 240 6513
124 409 10634
528 424 22787
384 637 11183
26 104 4254
313 654 30326
303 107 21170
179 245 31153
388 494 32101
148 276 16929
237 93 100
45 72 1694
625 340 15932
101 404 10804
593 577 25459
638 231 20315
407 125 23759
581 413 15495
310 451 17297
11 595 3279
374 561 29071
83 408 1880
652 334 18544
626 501 3776
323 632 24186
259 470 30154
368 321 2223
312 141 14574
613 357 27948
420 300 13472
616 420 14856
328 461 32365
319 331 7206
282 536 6970
211 266 28343
46 112 30535
141 24 10077
406 565 21711
534 143 3117
537 543 27730
313 520 6150
377 511 17900
192 426 26357
161 524 30030
277 173 26527
4 341 27180
96 368 12261
214 280 1346
195 508 576
543 315 9784
64 479 7817
406 634 279
415 261 13348
414 17 18946
194 210 1228
10 76 15476
438 314 20082
220 311 16189
389 431 14312
312 371 30940
131 584 3920
613 133 8223
376 531 31375
233 345 17158
616 623 6442
357 500 12890
229 213 21593
524 534 19069
306 437 1342
180 283 599
174 359 624
169 189 3959
269 469 23915
262 629 6837
400 133 28026
180 3 761
308 416 2362
175 300 5121
23 65 20679
127 268 29995
538 259 31277
51 350 30902
54 170 32443
482 217 28152
83 259 9030
394 145 14710
464 8 28285
518 452 19433
634 329 2011
377 271 11658
281 118 32480
445 46 17731
328 203 28056
515 185 23447
644 204 6801
385 399 10078
478 385 32212
573 591 1698
410 522 10626
165 652 28744
290 413 25112
355 29 15174
438 594 31637
620 141 7833
174 437 10221
208 431 22836
539 447 6284
190 604 21400
290 91 30693
639 453 21738
82 280 8496
242 360 9103
290 512 27628
15 91 11703
205 107 11712
446 506 12395
387 469 4303
371 602 21486
532 217 21916
619 10 27287
425 572 24332
466 41 31593
558 313 19505
11 408 5145
41 327 14044
516 228 5697
195 94 16167
112 218 25068
387 600 11134
580 253 29643
121 158 25033
323 40 22432
413 23 10693
322 550 16341
198 7 20916
506 205 14640
418 141 17892
10 160 24097
517 35 16578
612 643 12067
312 474 15479
204 207 24097
112 416 4153
94 274 30949
163 133 29397
62 270 756
581 275 18735
161 192 18791
267 478 661
128 434 7073
328 46 7744
452 17 10741
457 18 7262
343 507 10130
506 39 25203
606 173 32598
630 519 19955
389 578 851
6 228 8552
463 266 3369
607 638 30590
37 192 6539
196 141 17606
65 490 4990
167 350 31771
238 482 21562
478 494 29411
164 416 25267
626 395 31143
166 650 18996
420 220 4789
326 621 401
380 16 5066
279 169 25821
247 489 8440
248 580 16807
82 304 7746
558 186 31270
223 57 17056
536 169 24037
476 284 12913
157 521 5831
2 77 16513
519 587 32635
625 613 31785
491 449 29885
371 301 22588
283 300 17651
54 11 18870
616 167 23046
605 41 3692
249 600 10125
71 271 11053
615 322 23451
445 147 28177
156 34 23258
271 272 11507
260 349 16558
536 631 24892
255 369 667
310 363 31120
545 611 549
599 416 3836
471 605 23259
360 112 28340
383 124 8790
96 646 27271
359 94 11850
482 647 29065
567 365 22787
309 167 24492
208 227 14044
370 210 3643
298 323 20376
544 246 18552
369 101 17570
77 103 11973
167 493 27042
556 214 6182
87 637 18152
87 396 29950
423 344 21907
81 326 26444
602 121 5007
132 263 994
66 399 25012
144 636 19779
