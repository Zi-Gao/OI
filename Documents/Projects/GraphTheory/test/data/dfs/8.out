1 192 126 426 223 57 276 105 120 518 270 149 469 281 118 80 432 439 254 275 285 452 17 522 87 242 101 236 404 360 112 218 368 321 416 616 167 350 609 79 493 420 220 106 235 311 300 358 631 382 607 610 90 43 199 344 562 445 46 570 336 450 3 525 41 327 430 486 84 180 193 283 481 376 316 642 531 133 47 8 122 582 143 119 145 386 576 638 231 147 117 597 293 593 577 578 255 241 563 34 387 600 499 76 366 369 9 203 411 634 329 153 436 56 93 539 177 414 7 488 354 574 553 67 228 282 536 169 189 306 437 413 23 65 490 253 103 116 268 289 468 30 14 515 185 221 624 161 92 524 277 173 534 447 123 357 500 68 260 52 349 594 48 449 423 252 443 109 239 75 312 141 24 424 561 261 474 190 504 174 359 94 274 195 467 315 50 487 383 124 409 96 646 476 284 171 146 158 58 64 86 479 232 18 440 168 380 16 535 471 605 618 505 508 211 266 573 591 604 649 371 301 19 361 278 494 265 648 602 121 556 214 273 280 580 318 655 381 395 623 396 233 345 611 637 375 240 603 442 352 181 183 251 187 